core/src/test/java/com/digitalpebble/stormcrawler/helper/initialisation/ClassWithoutValidConstructor.java

```diff
@@ -0,0 +1,9 @@
+package com.digitalpebble.stormcrawler.helper.initialisation;
+
+import com.digitalpebble.stormcrawler.helper.initialisation.base.AbstractClass;
+
+public class ClassWithoutValidConstructor extends AbstractClass {
+    public ClassWithoutValidConstructor(int i) {
+        System.out.println(i);
```

Comment:
What is `i` and why sysout? Might be difficult to check against sysout in unit tests ;)